strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NpmPackageGenerator.java

```diff
@@ -66,6 +76,18 @@ public NpmPackageGenerator in(Path path)
         return this;
     }
 
+    @Override
+    public Path generateArtifact(URI uri,
+                                 int size)
+            throws IOException
+    {
+        new NpmPackageGenerator(Paths.get(uri));
```

Comment:
why you need this here?

Reply:
I had some misunderstanding. Removed the same.